scripts/prepare_node_config.sh

```diff
@@ -25,8 +25,14 @@ then
     exit 1
 fi
 
+if [ -z "$SYSTEM_RESERVED_MEMORY" ]
+then
+    SYSTEM_RESERVED_MEMORY="21"
+fi
+
 echo "NODE_TYPE=$NODE_TYPE" > $CONFIG_OUTPUT_FILE
 echo "NODE_NAME=$NODE_NAME" >> $CONFIG_OUTPUT_FILE
 echo "NODE_CONTROL_PLANE_VIP=$NODE_CONTROL_PLANE_VIP" >> $CONFIG_OUTPUT_FILE
 echo "NODE_CONTROL_PLANE_PORT=$NODE_CONTROL_PLANE_PORT" >> $CONFIG_OUTPUT_FILE
 echo "NODE_JOIN_TOKEN=$NODE_JOIN_TOKEN" >> $CONFIG_OUTPUT_FILE
+echo "SYSTEM_RESERVED_MEMORY"=$SYSTEM_RESERVED_MEMORY >> $CONFIG_OUTPUT_FILE
```

Comment:
Let's keep it consistent with how default values are handled in `scripts/prepare_cluster.sh`.
```suggestion
echo "NODE_TYPE=$NODE_TYPE" > $CONFIG_OUTPUT_FILE
echo "NODE_NAME=$NODE_NAME" >> $CONFIG_OUTPUT_FILE
echo "NODE_CONTROL_PLANE_VIP=$NODE_CONTROL_PLANE_VIP" >> $CONFIG_OUTPUT_FILE
echo "NODE_CONTROL_PLANE_PORT=$NODE_CONTROL_PLANE_PORT" >> $CONFIG_OUTPUT_FILE
echo "NODE_JOIN_TOKEN=$NODE_JOIN_TOKEN" >> $CONFIG_OUTPUT_FILE
echo "SYSTEM_RESERVED_MEMORY"=${SYSTEM_RESERVED_MEMORY:-21} >> $CONFIG_OUTPUT_FILE
```

Reply:
I have also done this for the NODE_TYPE. To be consistent 😉